source/common/router/header_parser.h

```diff
@@ -38,16 +41,16 @@ class HeaderParser : public Http::HeaderEvaluator {
    */
   static HeaderParserPtr
   configure(const Protobuf::RepeatedPtrField<envoy::config::core::v3::HeaderValue>& headers_to_add,
-            bool append);
```

Comment:
Forget to remove the comment for `append` at https://github.com/envoyproxy/envoy/pull/22723/files#diff-b5669de72ffdab01ee2e54f54e9cb97f9d6f7bf4e053fb31c6af1ee56f2cf316R39

Reply:
right. Thanks.